.travis.yml

```diff
@@ -42,7 +42,6 @@ script:
   # Run tests
   # Publish image to registry if merged into master.
   - make test && ./release.sh
-  - make lint-examples
```

Comment:
@yankeexe Oh my god. Did you just resolve the merge conflict by removing everything that @kabirbaidhya did in #38 ?

Are you aware of the changes that is happening elsewhere? Do you know how to merge conflicts? Don't just remove code that you are not aware of.

🤕 🤦‍♂

Reply:
@yankeexe Before you merge conflicts, you should also see what has been pushed upstream so that you can understand the changes so that you could decide what you need to do.

Reply:
@mesaugat  @kabirbaidhya I didn't remove them. They are left to be committed. I unstaged them when I pulled from upstream. 
![image](https://i.imgur.com/z1hExWj.png)